scripts/debug_recording.py

```
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+"""
+Debug specific recording annotations.
+"""
+import sys
+
+import requests
+
+API_BASE_URL = "https://bsmarker.utia.cas.cz/api/v1"
+USERNAME = "remisvoj@cvut.cz"
+PASSWORD = "bsmarker20250903"  # pragma: allowlist secret
```

Ruff (0.14.7)

Possible hardcoded password assigned to: "PASSWORD"

(S105)